viz_scripts/timeseries_analysis.ipynb

```diff
@@ -269,7 +214,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Generate Timeseries Plots"
+    "## Generate Timeseries Plots"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Emissions per week"
```

Comment:
again, we should split out these study plots and the program-specific plots into separate notebooks.

Reply:
Split notebooks by program/not program. There are now generic metrics/timeseries notebooks and mode-specific metrics/timeseries notebooks. There is also energy_calculations as its own notebook.